mathics/core/symbols.py

```diff
@@ -575,7 +575,7 @@ def get_attributes(self, definitions):
         return definitions.get_attributes(self.name)
 
     def get_name(self, short=False) -> str:
-        return self.name
+        return self.name.split("`")[-1] if short else self.name
```

Comment:
@rocky, notice that in the code, we use the method `Definitions.shorten_name()` which keeps track of the current `$Context` and `$ContextPath` to decide whether the name of a symbol should or should not be shortened.

Reply:
Ok. I am not totally certain how to use this here or if this is the right thing, user interface-wise.

 It might be that in the debugger TraceActivate and TraceDebug do some sort of context-based searching. But on the other hand , it might also be that in a debugger ContextPath's change so you want the looseness. When the test is done might be in a totally different context from the context that set up the trace. 

I will add some comments and possibly TODO's in the debugger code so this idea not lost and made more in the forefront. 

But I also note that if there is a parameter named "short=False", it is wrong to have it not work.

Reply:
> 
> But I also note that if there is a parameter named "short=False" offered, it is wrong to have it not work.

Yep, I totally agree. Let's merge this when you feel it is ready.